sdk/servicebus/service-bus/src/receiver.ts

```diff
@@ -367,13 +369,14 @@ export class SessionReceiver {
   }
 
   /**
-   * Renews the lock on the session.
-   * Check the `sessionLockedUntilUtc` property on the reciever for the time when the lock expires.
+   * Renews the lock on the session for the duration as specified during the Queue/Subscription
+   * creation. Check the `lockedUntilUtc` property on the Service Bus entity client for the time when the lock expires.
```

Comment:
> Check the `lockedUntilUtc` property on the Service Bus entity client for the time when the lock expires.

- `lockedUntilUtc` -> `sessionLockedUntilUtc`
- `property on the Service Bus entity client` -> `property on the SessionReceiver`

T